cpp/ps0000/p0000_solution.cpp

```diff
@@ -1,7 +1,8 @@
-#include <iostream>
+#include <iostream> // 2/2
 using namespace std;
-int main()
+int main() // 1/1
 {
-	cout << "Hello, World!\n";
-  return 0;
-}
\ No newline at end of file
+	cout << "Hello, World!\n"; // 4/4
+	return 0; // 1/1
+}
+// Excellent work 😎 10/10 ⭐

```

Comment:
- [x] all code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 bonus marks for your first commit